packages/xroad-monitor-collector/redhat/SPECS/xroad-monitor-collector.spec

```diff
@@ -9,7 +9,7 @@ Release:            %{rel}%{?snapshot}%{?dist}
 Summary:            X-Road Monitoring Data Collector
 Group:              Applications/Internet
 License:            MIT
-Requires:           systemd, java-1.8.0-openjdk
+Requires:           systemd, java-1.8.0-openjdk, xroad-common
```

Comment:
maybe add cronie dependency (as is in xroad-catalog). cronie-anacron however seems unnecessary?